demos/common/include/iot_config_common.h

```diff
@@ -155,6 +156,16 @@
     #define IOT_NETWORK_RECEIVE_TASK_STACK_SIZE    ( 5 * configMINIMAL_STACK_SIZE )
 #endif
 
+/* Default iot thread stack_size. */
+#ifndef IOT_THREAD_DEFAULT_STACK_SIZE
+    #define IOT_THREAD_DEFAULT_STACK_SIZE    configMINIMAL_STACK_SIZE
```

Comment:
We should be careful about using the minimal stack size as the default (although I don't have a good default value).

Reply:
I only matched the previous behavior

Reply:
For now I am leaving this as minimal, it is the best on resources and if any platform needs more stack space it should be overridden for that platform.  We can always increase this configuration parameter later.